spec/javascripts/Policies/components/__snapshots__/HeaderButton.spec.jsx.snap

```diff
@@ -1,21 +1,59 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`should be able to render a "cancel" button variant 1`] = `
-<div
-  class="PolicyChain-addPolicy--cancel"
+<button
+  class="pf-c-button pf-m-link PolicyChain-addPolicy--cancel"
+  type="button"
 >
-  <i
-    class="fa fa-times-circle"
+  <span
+    class="pf-c-button__icon"
+  >
+    <svg
+      aria-hidden="true"
+      fill="currentColor"
+      height="1em"
+      role="img"
+      style="vertical-align:-0.125em"
+      viewBox="0 0 352 512"
+      width="1em"
+    >
+      <path
+        d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"
+        transform=""
+      />
+    </svg>
+  </span>
+  <span
+    class="pf-c-button__text"
```

Comment:
This works correctly?

Reply:
Don't worry, this is generated automatically, only used for js tests with jest

Reply:
This is correct, @thomasmaas. Any link-like button (`pf-m-link`) will have this class in its text.

Reply:
However, I don't think this snapshots make sense anymore @damianpm. The icon class used to be readable but now being it a SVG it doesn't say anything. I'd rewrite the test and look for a `PlusIcon` and `TimesIcon` directly.

Reply:
@josemigallas let's update the test in a new PR, as it's out of scope of this one

Reply:
Fair enough